### PR TITLE
CI AI-392

### PR DESCRIPTION
**Pull request by jackkav**
### Summary
CI can stand up everything and check collector comes up ok with "ct install", essentially connectivity checks.

### Description

Helm installs kong and collector in order to check new PRs for configuration and connectivity issues

### TODO
- [x] pushed a copy of 1.5.0.0-alpine to kong docker registry
- [x] use bitnami for parity with kong chart
- [x] set up kind
- [x] create k8s state
- [x] install kong
- [x] use "ct install" to test that helm is able to run collector
- [x] override registry location in default-values.yaml

### Notes for reviewers
- Any thing I can do about this Release step always being yellow?
- As discussed with @hbagdi I believe we are still planning to merge this repo into kong/charts upon collector releases, but CI will take place here unless we find a way to make it work effectively in kong/charts

Based on https://github.com/helm/chart-testing/blob/master/examples/kind/test/e2e-kind.sh
With additional prereqs

### CODE REVIEW

File: charts/kong-collectorapi/ci/default-values.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+image:
+  repository: registry.kongcloud.io/kong-brain-immunity
```
Comment (hbagdi) [inline charts/kong-collectorapi/ci/default-values.yaml:2]: This CI should use Bintray, the source our customers use for brain and immunity.

File: charts/kong-collectorapi/requirements.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   version: 8.2.1
```
Comment (hbagdi) [inline charts/kong-collectorapi/requirements.yaml:3]: Sync this version up from Kong's chart. Keeping them the same helps avoid any discrepancies.

**Comment by hbagdi** [comment]
At some point, I think we should be able to move this workflow into `kong/charts` as well. There should be a way to tell `ct` to process only specific charts in the `charts` repository.

File: charts/kong-collectorapi/ci/default-values.yaml (hunk 1 above)
Comment (jackkav) [reply charts/kong-collectorapi/ci/default-values.yaml]: oh this didn't work before I'll give it another try

Comment (jackkav) [reply charts/kong-collectorapi/ci/default-values.yaml]: it seems as if the bintray and kind incompatibility is still an issue. This is why I'm using kongcloud registry.
image
https://www.jfrog.com/jira/browse/RTFACT-20170
it says resolved in January but I remember this not working last time i tried it February too.

Comment (jackkav) [reply charts/kong-collectorapi/ci/default-values.yaml:2]: I can try to workaround the workaround with kind load I guess

**Pull request merged**